manager/manager.go

```diff
@@ -126,6 +126,13 @@ type Config struct {
 	// first node in the cluster, this setting is used to set the cluster-wide mandatory
 	// FIPS setting.
 	FIPS bool
+
+	// DefaultAddrPool specifies default subnet pool for global scope networks
+	DefaultAddrPool []*net.IPNet
```

Comment:
Can we change this to a slice of strings here and everywhere else?

Reply:
Because it is derived from CLI and can be error prone due to human interaction, I was told to have well defined structure when we use API calls between CLI--> MOBY-->SWARMKIT-->LIBNETWORK. We do validation at CLI and convert string info into IPnet so that we will not have any error while parsing string deep inside stack.  Thats the main reason we have IPnet. earlier I was having same string all along the way and parsed only inside swarmkit. But after discussion with @thaJeztah, I reworked on the info we are passing in API calls. I hope this is ok with you.

Reply:
ahh I see. Sorry I missed the previous discussion around this. Is it not possible to validate it and keep it string?

Reply:
I was ok with that. @thaJeztah advised to keep API params neat especially when we communicate from one library to other. I had multiple rounds of discussion with him in private chat :).  Then I agreed to change it.

Reply:
Posted on slack, but let me put it here as well @anshulpundir is there a specific concern you have with using `[]*net.IPNet` here?

Reply:
discussion from slack:
thajeztah [11 hours ago
What's your biggest concern with the `[]*net.IPNet` instead of a `[]string` (wondering)


anshul [4 hours ago]
Simplicity. The addresses are stored as strings in the cluster object and so I was wondering if they can be just passed around as strings too. 

BTW Are you concerned about corruption in transfer?


thajeztah [4 hours ago]
The main reason I came with the `net.IPNet` was for the CLI side of things, where we had a lot of custom parsing stuff.
Originally the design was passing the literal format that's used for the flag input around, and validation/parsing didn't happen until we reached swarmkit; which didn't make sense to me; I'm for passing around wel-formed values (independent from how it's passed on the CLI); API's are not for humans, they're for machines.


anshul [9 minutes ago]
that makes sense. I guess it seems a little odd to me that we parse from string to net.IPNet and then pass it as that, only to convert it back to string. (edited)


anshul [9 minutes ago]
Maybe we can do the validation an keep it string?

Reply:
@Selansen would this complicate things for you, or easy to change?

Reply:
thats kind of going back to my old initial code. Need re write it again :)

Reply:
The original implementation passed a single string; I think @anshulpundir meant to use; `[]string`, where each string is "`net.IPNet.String()`"

(@anshulpundir if you can confirm)